### DISCUSSION TITLE
Track Changes with WOPI

### DISCUSSION THREAD

**Discussion by kucuk-furkan** [petrsvihlik/WopiHost#160] [Q&A]
Hello,

We're using OOS with WOPI integration. We want to enable track changes like office for web. But we couldn't find how to do it. Is it possible? If is it possible how can we archive it?

**Comment by petrsvihlik** [comment]
I have no idea. I've seen some posts in MS forums indicating that multiple folks are facing this problem. It might not be supported for WOPI integrations.
